packages/muze/src/canvas/component-wrapper-maker.js

```diff
@@ -81,8 +82,7 @@ const createHeaderWrapper = (headerType, layoutManager, renderDetails) => {
         });
 
         if (headerConfig.align === CENTER) {
-            headerConfig.alignment = null;
-            headerConfig.alignWith = null;
+            headerConfig.alignment = HORIZONTAL_CENTER;
```

Comment:
Remove the if conditions and instead store a strategy/map instead for this kind of code.